.pre-commit-config.yaml

```diff
@@ -7,6 +7,7 @@ repos:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
     -   id: check-yaml
+        args: [--allow-multiple-documents]
```

Comment:
What is this used for?

Reply:
@sutaakar This check-yaml arg update allows to use multiple documents in single yaml file during pre-commit check (minio-deployment.yaml)